luzer/tests/test_lib.c

```diff
@@ -0,0 +1,15 @@
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+#define MESSAGE "Lua"
+
+int
+say_hello(const char *buf) {
```

Comment:
This function is almost the same as in the previous commit. May we declare it non-static and use this library via `ffi.load()` as well?